database/clickhouse/clickhouse.go

```diff
@@ -80,6 +80,11 @@ func (ch *ClickHouse) Open(dsn string) (database.Driver, error) {
 		return nil, err
 	}
 
+	database := purl.Query().Get("database")
+	if db := strings.TrimPrefix(purl.Path, "/"); len(database) == 0 && len(db) != 0 {
```

Comment:
Remove this. It's more confusing if there are 2 ways to specify the same thing. e.g. if the values differ, which one should take precedence?

Reply:
This is necessary for compatibility with both DSN formats in V1 and V2